website/src/pages/docs/Theme/ThemeTokens.mdx

```diff
@@ -96,8 +96,22 @@ const theme = createTheme({
 });
 ```
 
+## Spacings
+
+Determine the spacing between objects such as `margin` and `padding`.
+
+```ts filename="kuma.config.ts"
+const theme = createTheme({
+  spacings: {
+    sm: "1em",
+    md: "4em",
+    lg: "8em",
+  },
+});
+```
+
 <Callout type="warning">
```

Comment:
We won't actually need this Callout anymore, as our theme tokens will be complete once this PR is merged 😉

Reply:
@poteboy According to #257, It seems that we still need `sizes`, `radii` and `zIndices`.

Reply:
Oops, my bad